src/test/java/calculator/CalculatorTest.java

```diff
@@ -0,0 +1,84 @@
+package calculator;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class CalculatorTest {
+	Calculator calculator = new Calculator();
+
+	@DisplayName("빈 문자열 또는 null 값을 입력할 경우 0을 반환")
+	@ParameterizedTest
+	@NullAndEmptySource
+	public void test_1(String input) throws Exception {
```

Comment:
```suggestion
	public void calculate(String input) {
```

메서드명도 자바 코딩 컨벤션을 지켜주세요
그리고 의미 있는 명칭을 쓰는걸 추천드립니다.
불필요한 exception은 제외하는게 좋겠네요

Reply:
네 수정하겠습니다.